consensus/consensus_v2.go

```diff
@@ -343,7 +341,7 @@ func (consensus *Consensus) Start(
 		}
 		consensus.getLogger().Info().Time("time", time.Now()).Msg("[ConsensusMainLoop] Consensus started")
 		defer close(stoppedChan)
-		ticker := time.NewTicker(3 * time.Second)
+		ticker := time.NewTicker(time.Second)
```

Comment:
I recommend leaving a comment as to why 1 sec ticker is chosen which was previously 3 seconds?

Reply:
I would keep the 3 seconds. I don't see clear good reason to shorten it now.

Reply:
I actually not sure. This ticker is to check the consensus or view change timeout. I guess EA want to speed up the checking of timeout of viewchange.